ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -226,6 +226,15 @@ static zend_string* sqlite_handle_quoter(pdo_dbh_t *dbh, const zend_string *unqu
 	if (ZSTR_LEN(unquoted) > (INT_MAX - 3) / 2) {
 		return NULL;
 	}
+	if(memchr(ZSTR_VAL(unquoted), '\0', ZSTR_LEN(unquoted)) != NULL) {
+		if (dbh->error_mode == PDO_ERRMODE_EXCEPTION) {
+			zend_throw_exception_ex(php_pdo_get_exception(), 0,
+				"SQLite PDO::quote does not support NULL bytes");
```

Comment:
We normally write `null bytes` in lowercase letters, please change it to lowercase here to be consistent (and below on line 234 too)